Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_Cryptomining.yml

```diff
@@ -212,7 +212,7 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
-    skipunavailable: false
+    skipunavailable: true
```

Comment:
shouldn't this be false?

Reply:
No. we've mentioned in the input description that it will be skipped.

Reply:
yes yes you are right :)